pandas/io/hdfs.py

```diff
@@ -0,0 +1,27 @@
+""" s3 support for remote file interactivity """
+from pandas import compat
+try:
+    import hdfs3
```

Comment:
you need to add hdfs3 on *some* builds (you can put on the same as s3fs) is fine in pandas/ci